its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CppTest.java

```diff
@@ -52,7 +51,7 @@ class CppTest {
   @Test
   void testCppOnly() throws Exception {
     String projectKey = "cpp";
-    String fileKey = TestUtils.hasModules(ORCHESTRATOR) ? "cpp:cpp:A8B8B694-4489-4D82-B9A0-7B63BF0B8FCE:ConsoleApp.cpp" : "cpp:ConsoleApp.cpp";
+    String fileKey = TestUtils.hasModules(ORCHESTRATOR) ? "cpp:cpp:A8B8B694-4489-4D82-B9A0-7B63BF0B8FCE:ConsoleApp.cpp" : "cpp:ConsoleApp/ConsoleApp.cpp";
```

Comment:
Is this because the base directory changed?
Is this change reflecting the behavior we want from the scanner?